packages/permission-log-controller/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "../../tsconfig.packages.json",
+  "compilerOptions": {
+    "baseUrl": "./"
+  },
+  "references": [
+    { "path": "../base-controller" },
+    { "path": "../controller-utils" },
+    { "path": "../json-rpc-engine" }
+  ],
+  "include": [
+    "../../types",
+    "./src",
+    "tests/PermissionLogController.test.ts",
+    "tests/helpers.ts"
```

Comment:
Nit: This works, but it might be nice to include everything in `tests` so we don't have to remember to update this list if we add anything new to this directory:

``` suggestion
    "./tests",